src/kaenguruen/load_data.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
+"""
+Script to parse Kaenguruen math problems from text files into a list of dictionaries.
+The dictionaries can be later dumped to JSON files.
+"""
+
+import os
+import json
+import argparse
+from pathlib import Path
+from typing import Dict, List, Any
+
+
+def parse_text_file(filepath: str, data_dir: str) -> Dict[str, Any]:
```

Comment:
```suggestion
def parse_text_file(filepath: str) -> Dict[str, Any]:
```
no need to have data dir as input